src/admin/user/user.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+import { TableBody } from '@material-ui/core'
+
+import { users } from 'admin/user/users'
+import { OneUser } from 'admin/user/one-user'
+
+export const User = () => <TableBody>
```

Comment:
Also, naming this component as `User` is a lie, since it holds a `<TableBody/>` containing all users. Let's rename this component to `<Body/>` and the former `<Body/>` component to `<Row/>`.

Reply:
Needless to say, the folders and file names also have to be changed.

Reply:
done